### PR TITLE
contrib/k8s: fix cluster role

### PR DESCRIPTION
**Pull request by hunchback**
To test:

```
kubectl create namespace mynamespace
cat skydive.yaml | sed -e "s/namespace: default/namespace: mynamespace/" | kubectl delete -n mynamespace -f -
```

### CODE REVIEW

File: contrib/kubernetes/README.md [hunk 1]
```diff
@@ -45,3 +45,9 @@ Change etcd port from 12379/12380 to 22379/22380:
 ```
 cat skydive.yaml | sed -e s/12379/22379/ -e s/12380/22380/ | kubectl apply -f -
 ```
+
+Deploy to namespace `mynamespace` insread of `default`:
```
Comment (masco) [inline contrib/kubernetes/README.md]: s/insread/instead

**Comment by masco** [comment]
run skydive-cdd-overview-tests
run skydive-functional-hw-tests
run skydive-functional-tests-backend-elasticsearch

**Comment by hunchback** [comment]
run skydive-cdd-overview-tests
run skydive-functional-hw-tests

**Comment by nplanel** [comment]
run skydive-functional-hw-tests

**Pull request merged**